carl/__init__.py

```diff
@@ -1,4 +1,7 @@
 __copyright__ = "Copyright 2021, AutoML.org Freiburg-Hannover"
 __license__ = "Apache-2.0 License"
-__version__ = '0.1'
-__author__ = 'Carolin Benjamins and Theresa Eimer, Aditya Mohan, Frederik Schubert, Bodo Rosenhahn, André Biedenkapp, Frank Hutter, Marius Lindauer'
+__version__ = "0.1"
+__author__ = (
```

Comment:
Those are paper authors, but not software authors, right? Would you keep everyone?

Reply:
I will follow Eddie's convention/advice